src/api/core/Models.ts

```diff
@@ -0,0 +1,6 @@
+import { Pet } from "../../models/Pet";
+import { Tutor } from "../../models/Tutor";
+
+const __MODELS__ = [Pet, Tutor];
```

Comment:
@bigLucas
In what context is the variable between underscore used?

Reply:
I just copied this code from onother project... It is not a name convention, we could choose any name.